test/test_jit.py

```diff
@@ -7812,8 +7812,8 @@ def test_passing_unknown_kwargs(self):
     def test_passing_and_returning_lists(self):
         # Replace with actual test once we support lists.
         a, b = torch.rand(5), torch.rand(5)
-        output = torch.ops.aten.cat([a, b])
-        output_ref = torch.cat([a, b])
+        output = torch.ops.aten.meshgrid([a, b])
```

Comment:
Why does this need to be changed?

Reply:
The old test doesn't work. I have two `cat` native functions in this PR and it will make the old test fail. I checked with @gchanan about this issue. We found torch.ops.aten doesn't work if you overload native functions.

Reply:
Basically, there's nothing special about the function we test here except that it takes a TensorList.  So, `meshgrid` works just as well as `cat`.  `cat` no longer works through this code path, which is testing user ops, which don't support overloads.